.github/workflows/build.yml

```diff
@@ -119,7 +119,7 @@ jobs:
   build-windows:
     needs: check-changes
     if: ${{ needs.check-changes.outputs.has_changes == 'yes' || github.event_name == 'push' }}
-    runs-on: [windows-2019]
+    runs-on: [ubuntu-latest]
```

Comment:
This github action still failed, could you double check it?

Reply:
It is because the dependent antrea-ovs image was not found, as I updated the tag from the original hash value to the script result of build-tag.sh .

Reply:
Here I also want to learn your thought about what is the expected version for Windows ovs images. If we switch to hpc as a base for Windows containers, we will not depend on the OS versions (e.g. 1809 or 2022) in Windows container and the tags. So shall we follow Linux container tag styles? Or shall we leave the OVS version (e.g., 3.0.5) in the tag? @XinShuYang @antoninbas  The same question exists for windows-base, which now only includes the cni files.

Reply:
OVS image is still built wrt ovs version and only downloaded in ovs/Dockerfile.windows (In this PR aftre using hpc image), IMO can prefer to keep the ovs version tag aloowing to identify ovs released version.

Reply:
One related ask is will we maintain multiple antrea-ovs windows images based on the OVS versions, e.g., ovs-3.0.5, ovs-3.1.0, etc. If no, I doubt it is meaningless to include OVS versions in the tag. BTW, the Linux images doesn't use OVS versions in the tag for antrea/openvswitch-xxx. So I'm thinking is it a chance to change Windows behavior?

Reply:
IMO, while using Antrea version as the tag offers straightforward alignment with Antrea releases, I believe that tagging our images based on the OVS version number would be more beneficial for several reasons:

1.Reduced Redundancy in Image Uploads: Antrea-Windows's release frequency often surpasses updates in OVS versions. If we continue tagging with the Antrea version, we need to upload new `antrea-ovs` image for each release, even when there are no updates to OVS. This process, despite being optimized on platforms like dockerhub, introduces unnecessary steps. Using the OVS version as a tag would eliminate this redundancy, only updating or uploading new images when there’s an actual change in OVS.

2.Streamlined Backporting Process: Our backporting efforts are generally driven by features or fixes specific to OVS versions. Using the OVS version as a tag allows us to quickly identify and work with the relevant image during backports, eliminating the need to double check the mapping relationship between Antrea and OVS versions. This can significantly accelerate our validation process when backporting changes.

3.Efficient Multi-branch Management: As we upgrade OVS across multiple branches, tagging with the Antrea version necessitates multiple image pushes to dockerhub, even if the target OVS image remains the same. By using the OVS version as our tag, we only need to upload a new version image when there's an actual update to OVS, reducing complexity and saving storage.